src/panels/lovelace/common/generate-lovelace-config.ts

```diff
@@ -413,6 +413,15 @@ export const generateLovelaceConfigFromData = async (
     viewEntities.length === 0 ||
     viewEntities[0].entity_id !== DEFAULT_VIEW_ENTITY_ID
   ) {
+    const hiddenPlatforms = ["mobile_app"];
+    const hidddenEntitites = entityEntries
+      .filter((entry) => hiddenPlatforms.includes(entry.platform))
+      .map((entry) => entry.entity_id);
+
+    hidddenEntitites.forEach((entity) => {
+      delete entities[entity];
+    });
```

Comment:
```suggestion
    const entities = entityEntries
      .filter((entry) => hiddenPlatforms.includes(entry.platform));
```

Reply:
Won't this just remove them from the area? It is still in states?

Reply:
Yes, that change will not work, but I'll see if I can make my current implementation simpler